Libraries/Text/RCTTextAttributes.m

```diff
@@ -73,6 +73,9 @@ - (void)applyTextAttributes:(RCTTextAttributes *)textAttributes
   _isHighlighted = textAttributes->_isHighlighted || _isHighlighted;  // *
   _tag = textAttributes->_tag ?: _tag;
   _layoutDirection = textAttributes->_layoutDirection != UIUserInterfaceLayoutDirectionLeftToRight ? textAttributes->_layoutDirection : _layoutDirection;
+  
+  // Transform
+  _textTransform = textAttributes->_textTransform ?: _textTransform;
```

Comment:
`Transform` can be considered as `Special`, so remove the comment.
Compare to additional special "undefined" value here.